src/components/10-atoms/input-text/index.js

```diff
@@ -216,13 +216,19 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
   }
 
   handleInput(ev) {
+    const { selectionStart = null } = ev.target;
+
     // sets this.invalidFormat
     this._formatCurrency(ev.target.value);
     this.onChange(ev, this.invalidFormat);
     // are we a 'controlled' input in the React sense?
     if (this.isControlled) {
       // yes, set UI from model state
       this.nativeInput.value = this.modelValue;
+
+      requestAnimationFrame(() => {
+        this.nativeInput.setSelectionRange(selectionStart, selectionStart);
```

Comment:
OK, this is a creative solution, that seems to be usable in all browsers we care about - for the narrow case of React controlled components with a trivial selection. Just curious: what if the selection is a non-trivial range spanning a few letters? Is it acceptable that the range is lost here? (I genuinely don't know).

Reply:
We would have to find out. That's why I wrote Marek directly that we have to observe the component. At the moment, this is the best fix.

Reply:
Ok, then good for now.